internal/kafka/command_topic_list.go

```diff
@@ -9,10 +9,18 @@ import (
 	"github.com/confluentinc/cli/v3/pkg/output"
 )
 
-type topicOut struct {
+type topicNameOut struct {
 	Name string `human:"Name" serialized:"name"`
 }
 
+type topicOut struct {
+	Name              string `human:"Name" serialized:"name"`
+	Kind              string `human:"Kind" serialized:"kind"`
+	IsInternal        bool   `human:"Is Internal" serialized:"is_internal"`
+	ReplicationFactor int32  `human:"Replication Factor" serialized:"replication_factor"`
+	PartitionsCount   int32  `human:"Partitions Count" serialized:"partitions_count"`
+}
```

Comment:
Why not print these new fields by default?

Reply:
yes lets do that

Reply:
(Awesome! Also note that `kafka topic describe` is already taken, and we can't change it until: https://confluentinc.atlassian.net/browse/CLI-2695?focusedCommentId=2026491)